contracts/modules/TokenExchanger.sol

```diff
@@ -82,9 +81,9 @@ contract TokenExchanger is OnlyOwnerModule {
      * @param _srcAmount The exact amount of source tokens to sell.
      * @param _minDestAmount The minimum amount of destination tokens required for the trade.
      * @param _expectedDestAmount The expected amount of destination tokens (used only in ParaSwap's Swapped event).
-     * @param _path Sequence of sets of weighted ParaSwap routes. Each route specifies an exchange to use to convert a given (exact) amount of a given
-     * source token into a given (minimum) amount of a given destination token. The path is a sequence of sets of weighted routes where the destination
-     * token of a set of weighted routes matches the source token of the next set of weighted routes in the path.
+     * @param _path Sequence of sets of weighted ParaSwap routes. Each route specifies an exchange to use to convert a given (exact) amount of
+     * a given source token into a given (minimum) amount of a given destination token. The path is a sequence of sets of weighted routes where
```

Comment:
Could this be breaking NatSpec? In any case, I find it a bit less readable than having each param description on a single line. Since solhint doesn't seem to be able to distinguish code length from comment length, should we maybe wrap NatSpec blocks that contain long lines within `/* solhint-disable */ ...   /* solhint-enable */` ?

Reply:
I don't see why the line length shouldn't be valid for comments as well and I would want to avoid disabling solhint unless we really have to. Natspec does support multiline comments so shouldn't be a problem https://solidity.readthedocs.io/en/latest/natspec-format.html#documentation-example